addons/mail/static/src/models/composer_view.js

```diff
@@ -520,7 +520,7 @@ registerModel({
                 default_model: this.composer.activeThread.model,
                 default_partner_ids: this.composer.recipients.map(partner => partner.id),
                 default_res_id: this.composer.activeThread.id,
-                mail_post_autofollow: true,
+                mail_post_autofollow: this.mailPostAutofollow,
```

Comment:
Hello,

Instead of passing this value around everywhere, I think you could just check for `this.composer.activeThread.hasWriteAccess` here and act accordingly?

At least as a quick fix until the day we can hopefully remove `mail_post_autofollow` altogether.

Reply:
Ok I'ill investigate this solution, thanks for the tip^^